### PR TITLE
fix session docs and controller

### PR DESCRIPTION
**Pull request by Npwskp**

### CODE REVIEW

**Review by vjumpkung — COMMENTED** [review]
Wait Hirasawa Au Approve

File: src/schema/session.schema.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export class SessionSchema {
 
   @Prop({ default: null, ref: () => UserSchema })
   @ApiProperty({
-    type: String,
+    type: UserSchema,
```
Comment (ishiasaka) [inline src/schema/session.schema.ts]: Use () => ... instead.

File: src/schema/session.schema.ts [hunk 2]
```diff
@@ -41,14 +41,14 @@ export class SessionSchema {
 
   @Prop({ default: null, ref: () => CouponSchema })
   @ApiProperty({
-    type: String,
+    type: CouponSchema,
```
Comment (ishiasaka) [inline src/schema/session.schema.ts]: Use () => ... instead.

File: src/schema/session.schema.ts (hunk 1 above)
Comment (Npwskp) [reply src/schema/session.schema.ts]: fixed

File: src/schema/session.schema.ts (hunk 2 above)
Comment (Npwskp) [reply src/schema/session.schema.ts]: fixed

**Review by lazykern — DISMISSED** [review]

File: src/schema/session.schema.ts [hunk 3]
```diff
@@ -41,14 +41,14 @@ export class SessionSchema {
...
-    type: String,
+    type: () => CouponSchema,
     nullable: true,
     description: 'Coupon ID',
   })
   coupon: Ref<CouponSchema>;
 
   @Prop({ required: true, ref: () => TablesSchema })
-  @ApiProperty({ type: String, description: 'Table ID' })
+  @ApiProperty({ type: TablesSchema, description: 'Table ID' })
```
Comment (lazykern) [inline src/schema/session.schema.ts]: Where's the `() =>`

Comment (Npwskp) [reply src/schema/session.schema.ts]: fixed

**Review by lazykern — APPROVED** [review]

**Pull request merged**